memory/hugepage_sanity.py.data/README.md

```diff
@@ -0,0 +1,20 @@
+Overview
+----------
+The hugepage_sanity test, tests the allocation of hugepages on Linux systems. It allocates a specified amount hugepages and then performs write operations to verify that the memory is accessible and functioning correctly. It primarily uses mmap system call and memset function to allocate the hugepages.
+
+
+Prerequisites
+--------------
+This test script requires a Linux system with hugepages support enabled. You must have the hugetlbfs supported in order to use hugepages.
+Additionally, you will need Python 3.x and the avocado-framework package installed on the Linux system.
+
+
+Parameters
+--------------
+* hpagesize : the size of hugepage in MB you want to run the test on, by default it is the default size supported on the kernel
+Example : hpagesize: 2
+This runs the test for hugepage size of 2MB
+
```

Comment:
can you also add if we need to take care of any range values?

Reply:
There is no such thing as range value here. I have addressed this by modifying the description :

`the size of hugepage in MB you want to run the test on (if system supports more than 1 hugepage size), by default it is the default size supported on the kernel`